benchopt/plotting/generate_html.py

```diff
@@ -98,6 +96,10 @@ def get_results(fnames, html_root, benchmark, config=None, copy=False):
             fname = fname_in_output
         fname = fname.absolute().relative_to(html_root.absolute())
 
+        custom_data, custom_dropdown = benchmark.get_plot_data(
+            df, plotly_style=True
+        )
+
```

Comment:
I would rather have an extra conversion step around here instead of having the in the plot generation.
```suggestion
        custom_data, custom_dropdown = benchmark.get_plot_data(df)
        custom_data = convert_plotly_style(custom_data)

```